client/src/API/board.js

```diff
@@ -1,9 +1,36 @@
 import axios from "axios";
 
-export const createBoard = (data) => {
+export const createBoard = async (data) => {
   try {
     const URL = `${window.location.origin}/api/board/`;
-    axios.post(URL, data);
+    return await axios.post(URL, data);
+  } catch (err) {
+    console.error(err);
+  }
+};
+
+export const getBoard = async (boardId) => {
+  try {
+    const URL = `${window.location.origin}/api/board/${boardId}`;
+    return await axios.get(URL);
+  } catch (err) {
+    console.error(err);
+  }
+};
+
+export const getBoardShallow = async (boardId) => {
+  try {
+    const URL = `${window.location.origin}/api/board/shallow/${boardId}`;
```

Comment:
just URL convention wise, it would probably be better to keep the api URL `${window.location.origin}/api/board/${boardId}` and just add a query parameter of `?shallow=true` or something similar